blocks/header/header.css

```diff
@@ -1,809 +1,290 @@
+.header__main-links {
+  top: var(--actions-links-height-mobile);
+  padding: 0;
+  height: calc(100vh - var(--actions-links-height-mobile));
 }
 
-header nav .sections .section.expand > a::after {
-  content: '\f104';
-  right: unset;
-  left: 0;
+.header__action-link {
+  display: flex;
+  flex-direction: column;
 }
 
-header nav .sections .section > ul > li.overview a::after {
-  content: unset;
+.header__action-link-text {
+  font-size: 12px;
+  letter-spacing: var(--f-caption-letter-spacing);
+  line-height: var(--f-caption-line-height);
 }
 
-/* 2nd and deeper levels */
-header nav .sections .section > ul > li.expand > ul {
-  padding-left: 15px;
+.header__mobile-actions {
+  display: flex;
+  gap: 16px;
 }
 
-header nav .sections .section > ul > li > ul > li a {
-  padding-top: 0;
+.header--hamburger-open .header__menu-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: rgb(20 20 20 / 40%);
+  width: 100vw;
+  height: 100vh;
+  display: block;
 }
 
-header nav .sections .section > ul > li.expand a::after {
-  content: '\f068';
+.header--hamburger-open .header__actions-list {
+  right: 0;
+  overflow: auto;
 }
 
-header nav .sections .section > ul > li.expand > ul li a::after {
-  content: unset;
+.header--hamburger-open .header__action-link-text {
+  display: inline;
 }
 
-header nav .sections .section a:hover {
-  color: currentcolor;
-  text-decoration: none;
+.header--hamburger-open .header__main-links {
+  right: 0;
 }
 
-header nav .sections .section .navigation-cta {
-  display: none;
+.header--hidden {
+  transform: translateY(-100%);
 }
 
-/* ----- tablet ------------------------------------------------------ */
-@media (min-width: 768px) {
-  /* the main mobile div */
-  header nav {
-    min-width: 756px;
-    flex: 0;
-  }
-
-  /* the search input  */
-  header nav .search > div {
-    width: 756px;
-    margin: 0 0 0 auto;
-  }
-
-  /* tools only use 50% of width */
-  header nav .tools {
-    width: 50vw;
-    right: -50vw;
-  }
-
-  header nav .hamburger-toggle[aria-expanded='true'] ~ .tools {
-    width: 50vw;
-    right: 0;
-  }
-
-  /* sections only use 50% of width */
-  header nav .sections {
-    width: 50vw;
-    right: -50vw;
-  }
-
-  header nav .hamburger-toggle[aria-expanded='true'] ~ .sections {
-    width: 50vw;
-    right: 0;
-  }
-}
-
-/* ------ Desktop -------------------------------------------------------- */
-@media (min-width: 992px) {
-  /* nav turns into css grid */
-  header nav {
-    flex: 1;
+@media (min-width: 1200px) {
+  .header {
     width: 100%;
-    display: grid;
-    grid-template:
-      '. brand tools .' 26px
-      '. brand search .' auto
-      '. . . . ' 27px
-      'sections sections sections sections' 48px
-      / calc(3.5% + 15px) auto auto calc(3.5% + 15px);
-    margin: 25px 0 0;
-    padding: 0;
+    justify-content: center;
   }
 
-  /* hide mobile search toggle */
-  header nav .search-toggle {
-    display: none;
+  .header__nav {
+    max-width: 1232px;
```

Comment:
Please use `var(--wrapper-width)` so all the items in the website have the same width

Reply:
Updated to `max-width: calc(var(--wrapper-width) + 32px);`

Reply:
Problem is that then the in page navigation needs to behave the same :(
If not, they look weird.